src/main/resources/i18n/messages.properties

```diff
@@ -861,49 +858,75 @@ project.samples.filter.by-endDate=To
-project.export.accession=NCBI Accession
-project.export.warning.select=One or more files must be selected for upload.
-project.export.submitter=Submitter
-project.export.status=Upload Status
-project.export.project=Project
-project.export.sidebar.title=SRA Submission
-project.export.table.title=Project Exports
-project.export.table.state=Upload State
-project.export.table.project=Project
-project.export.table.samples=Exported Samples
-project.export.warning.nofiles=Warning: Sample {0} has no files.  It cannot be submitted to NCBI.
-
 NcbiExportTable.loader-error=A project id is required.
 NcbiExportDetailsView.loader-error=No project id or export id.
 
+# TYPE: NcbiSubmissionRequest (webapp/resources/js/apis/export/ncbi.ts)
+NcbiSubmissionRequest.projectId=BioProject ID
+NcbiSubmissionRequest.projectId.description=BioProject to submit files to.  This project must be created in NCBI prior to this submission.
+NcbiSubmissionRequest.organization=Organization
+NcbiSubmissionRequest.organization.description=Name of organization submitting these samples.
+NcbiSubmissionRequest.namespace=Identifier Namespace
+NcbiSubmissionRequest.namespace.description=Prefix to use for submission identifiers in NCBI.
+NcbiSubmissionRequest.releaseDate=Release Date
+NcbiSubmissionRequest.releaseDate.description=Submission will not be made public until after the chosen release date.
+
+# TYPE: NcbiBioSample
+NcbiBioSample.id=BioSample ID
+NcbiBioSample.accession=NCBI Accession
+NcbiBioSample.bioSample=BioSample
+NcbiBioSample.libraryName=Library Name
+NcbiBioSample.libraryStrategy=Library Strategy
+NcbiBioSample.librarySource=Library Source
+NcbiBioSample.libraryConstructionProtocol=Library Construction Protocol
+NcbiBioSample.instrumentModel=Instrument Model
+NcbiBioSample.librarySelection=Library Selection
+NcbiBioSample.status=Status
+NcbiBioSample.singles=Single Files
+NcbiBioSample.pairs=Paired Files
+
+# TYPE: NcbiSubmission
+NcbiSubmission.id=Submission ID
+NcbiSubmission.project=Project
+NcbiSubmission.state=Status
+NcbiSubmission.submitter=Submitter
+NcbiSubmission.createdDate=Created Date
+NcbiSubmission.organization=Organization
+NcbiSubmission.bioProject=BioProject
+NcbiSubmission.ncbiNamespace=NCBI Namespace
+NcbiSubmission.releaseDate=Release Date
+NcbiSubmission.bioSamples=Samples
+
+# NEW
+CreateNcbiExport.title=NCBI SRA Export Tool
+CreateNcbiExport.details=Submission Details
+CreateNcbiExport.samples=Samples
+CreateNcbiExport.pairs=Paired End Sequencing Files
+CreateNcbiExport.singles=Single End Sequencing Files
+CreateNcbiExport.files.required=Must select a sequencing file.
+CreateNcbiExport.required=This is a required field.
+CreateNcbiExport.submit=Submit
+CreateNcbiExport.default.title=Default Sample Settings
+CreateNcbiExport.default.description=Values selected here will be used as default values for all samples.  Individual samples can can be overwritten.
```

Comment:
Change this to `Values selected here will be used as default values for all samples.  Individual samples can overwrite these default values.`

Reply:
Updated in 4a523a80f14e29bffd842d327ea840b68dcc89e9